lesson8-UnitTesting/FizzBuzz/src/FizzBuzz.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace FizzBuzz
+{
+    public class FizzBuzzClass
+    {
+        public static string FizzBuzzResult(int x)
```

Comment:
Naming here also can be improved:
- `FizzBuzzResult` -> `Generate`
- `x` -> `input` or `number`.

Reply:
done